README.md

```diff
@@ -11,8 +11,18 @@ This is the Java driver for Amazon Quantum Ledger Database (QLDB), which allows
 
 Version 1.x is still supported.
 
+For getting started with the driver, see [Java and Amazon QLDB](https://docs.aws.amazon.com/qldb/latest/developerguide/getting-started.java.html).
+
+## Requirements
+
+### Basic Configuration
+
+See [Accessing Amazon QLDB](https://docs.aws.amazon.com/qldb/latest/developerguide/accessing.html) for information on connecting to AWS.
+
 ## Getting Started
 
+Please see the [Quickstart guide for the Amazon QLDB Driver for Java](https://docs.aws.amazon.com/qldb/latest/developerguide/driver-quickstart-java.html).
+
 1. **Sign up for AWS** &mdash; Before you begin, you need an AWS account. For more information about creating an AWS 
 account and retrieving your AWS credentials, see [Accessing Amazon QLDB](https://docs.aws.amazon.com/qldb/latest/developerguide/accessing.html) in the Amazon QLDB Developer Guide.
 1. **Minimum requirements** &mdash; To use the Amazon QLDB Driver for Java, you'll need **Java 1.8+**. For more 
```

Comment:
Hmm - here's the min Java version, seems like it should be under requirements.

Reply:
Moved to requirements